interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -23,6 +24,11 @@ function __construct($template_mod = "general") {
       $this->assign("gbl_vitals_options",$GLOBALS['gbl_vitals_options']);
     }
 
+		function setFormId($form_id){
```

Comment:
To clarify my above comment (it's getting late), thank you for this code contribution :)